ng2-components/ng2-activiti-form/src/components/activiti-form.component.spec.ts

```diff
@@ -634,4 +634,74 @@ describe('ActivitiForm', () => {
         expect(visibilityService.refreshVisibility).toHaveBeenCalledWith(field.form);
     });
 
+    it('should load form for ecm node', () => {
+        spyOn(formComponent, 'loadFormForEcmNode').and.stub();
+
+        formComponent.nodeId = '<id>';
+        formComponent.ngOnInit();
+
+        expect(formComponent.loadFormForEcmNode).toHaveBeenCalled();
```

Comment:
I think that complicates testing. This test checks `ngOnInit` and `nodeId` condition.